### ISSUE TITLE
downsampling return should use same padding not valid

### ISSUE THREAD

**Issue by Lapis-Hong** [zonetrooper32/VDCNN#12]
return tf.layers.conv1d(
            inputs=pool, filters=pool.get_shape()[2] * 2, kernel_size=1,
            strides=1, padding='same', use_bias=False)
